.github/workflows/publish-commit-bottles.yml

```diff
@@ -311,22 +346,21 @@ jobs:
           done
 
           # One last check...
-          if [[ -z "$success" ]] && [[ "$EXPECTED_SHA" != "$(git ls-remote origin "pull/${{inputs.pull_request}}/head" | cut -f1)" ]]
+          if [[ -z "$success" ]] && [[ "$EXPECTED_SHA" != "$(git ls-remote origin "pull/$PR/head" | cut -f1)" ]]
           then
             echo "::error ::No attempts remaining. Giving up."
             exit 1
           fi
 
       - name: Approve PR and enable automerge
-        if: fromJson(steps.pr-branch-check.outputs.bottles)
         id: automerge
         env:
-          GH_TOKEN: ${{secrets.HOMEBREW_GITHUB_PUBLIC_REPO_TOKEN}}
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
This will help fix https://github.com/Homebrew/homebrew-core/pull/128387#issuecomment-1509162706.